flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,65 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
+	{
+		var arrayNames:Array<FlxAnimation> = [];
+		
+		for (i in _animations)
+		{
+			arrayNames.push(i);
+		}
+
+		return arrayNames;
+	}
+
+	public function nameList():Array<String>
+	{
+		var anims = animationList();
+		var names:Array<String> = [];
+		for (i in 0...anims.length)
+		{
+			names.push(anims[i].name);
+		}
+
+		return names;
+	}
+
+	public function exists(name:String):Bool
+	{
+		#if (haxe > "4.0.5")
+		return nameList().contains(name);
+		#else
+		for (i in 0...nameList().length)
+		{
+			if (nameList[i] == name)
+				return true;
+		}
+
+		return false;
+		#end
```

Comment:
this should just be
```hx
return _animations.exists(name);
```

Reply:
Thanks, I'm just so stupid that I can't code properly.
Resolved